language/movie_nl/main_test.py

```diff
@@ -69,18 +69,18 @@ def test_process_movie_reviews():
     entities = [json.loads(entity) for entity in entities]
 
     # assert sentiments
-    assert sentiments[0].get('sentiment') == 1.0
+    assert sentiments[0].get('sentiment') == 0.9
```

Comment:
Yes but they don't happen that frequently. And so above check allows us to see if something really failed.  Non-None will not allow us to detect some issue model or API might have.